pages/index.js

```diff
@@ -37,22 +37,18 @@ export default function Home({ days }) {
   }
 
   return (
-    <div>
-      <main>
-        <CardContainer>
-          {days.map((day) => (
-            <Card
-              key={day.id}
-              id={day.id}
-              weight={day.weight}
-              date={day.date}
-              height={day.height}
-              feastTime={day.feastTime}
-              handleDelete={handleDelete}
-            />
-          ))}
-        </CardContainer>
-      </main>
-    </div>
+    <CardContainer>
+      {days.map((day) => (
+        <Card
+          key={day.id}
+          id={day.id}
+          weight={day.weight}
+          date={day.date}
+          height={day.height}
+          feastTime={day.feastTime}
```

Comment:
I would recommend to pass down the whole {day} object instead of every single attribute. Makes the code i bit lighter :)

Reply:
yeah, right. I just did that, thanks!!